rllib/examples/rock_paper_scissors_multiagent.py

```diff
@@ -202,11 +183,14 @@ def entropy_policy_gradient_loss(policy, model, dist_class, train_batch):
         loss_fn=entropy_policy_gradient_loss)
     EntropyLossPG = PGTrainer.with_updates(
         name="EntropyPG", get_policy_class=lambda _: EntropyPolicy)
-    run_heuristic_vs_learned(use_lstm=True, trainer=EntropyLossPG)
+    run_heuristic_vs_learned(args, use_lstm=True, trainer=EntropyLossPG)
 
 
 if __name__ == "__main__":
-    # run_same_policy()
-    # run_heuristic_vs_learned(use_lstm=False)
-    run_heuristic_vs_learned(use_lstm=False)
-    # run_with_custom_entropy_loss()
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--stop", type=int, default=1000)
```

Comment:
Consider keeping parser args at the top of the file by convention.

Reply:
👍

Reply:
Done.